webpack/webpack.config.base.js

```diff
@@ -218,8 +221,12 @@ const config = {
     }),
 
     new webpack.EnvironmentPlugin({
+      COINBASE_APP_ID: process.env.COINBASE_APP_ID ?? '',
+      MOONPAY_API_KEY: process.env.MOONPAY_API_KEY ?? '',
```

Comment:
Isn't the point of adding the dotenv config so we can remove the environment plugin completely?

Reply:
`dotenv` loads variables from .env locally into `process.env`: https://github.com/motdotla/dotenv

However, you still need a webpack plugin for builds. As an alternative, we could use: https://github.com/mrsteele/dotenv-webpack

Reply:
If we keep using `EnvironmentPlugin` with .env we should change our implementation bc it is meant to just be an array or an object with a fallback if there is no variable defined in the loaded .env.

https://webpack.js.org/plugins/environment-plugin/

Reply:
With dotenv-webpack, can env vars be referenced in our code without explicitly referencing them in the environmentplugin?

Reply:
Yep, that is correct.